src/Decompiler/Analysis/IndirectCallRewriter.cs

```diff
@@ -54,11 +58,13 @@ public class IndirectCallRewriter
             this.expander = new IndirectCallExpander(ssa);
             this.ssaIdTransformer = new SsaIdentifierTransformer(ssa);
             this.eventListener = eventListener;
+            this.emiter = new ExpressionEmitter();
         }
 
-        public void Rewrite()
+        public bool Rewrite()
```

Comment:
Please add a summary comment explaining what is being rewritten, and what it means when true/false is returned.